src/css/index.css

```diff
@@ -91,3 +91,8 @@ input:focus {
   display: flex;
   gap: 4px;
 }
+
+.sendbird-modal-root {
+  position: absolute;
+  z-index: 2147483647;
```

Comment:
can we replace with MAX_Z_INDEX?

Reply:
Oh we can't import the module(js file; const.ts) in .css file 🥲